src/plugin.ts

```diff
@@ -8,8 +8,9 @@ import normalizeNode from './normalize'
 import { isInline, isVoid } from './interfaces'
 
 /** A Slate editor augmented with support for editing CNX documents. */
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface CnxEditor extends Editor {
+    /** Can {@code element} be used as content of an {@link Equation}? */
+    isEquationContent(element: Element): boolean
```

Comment:
This comment doesn't seems really clear to me.
btw. `<equation>` can be directly followed be `<math>`, right?
```html
<equation>
   <math>...</math>
</equation>
```